### DISCUSSION TITLE
ShapeletTransformClassifier ressources

### DISCUSSION THREAD

**Discussion by cedced19** [alan-turing-institute/sktime#2728] [Q&A]
Hi,

I want to understand the resources linked to [`ShapeletTransformClassifier`](https://www.sktime.org/en/stable/api_reference/auto_generated/sktime.classification.shapelet_based.ShapeletTransformClassifier.html#shapelettransformclassifier).

`RandomShapeletTransform` and `RotationForest`, or other algorithms like those one, are not explicitly named in *Jon Hills et al.* and *A. Bostrom and A. Bagnall* articles.

Any chance that it is not the good one referenced here ?

Can someone explain me? 

Thank you for your help.

**Comment by fkiraly** [comment]
@cedced19, are you saying that the reference in the docstring is incorrect?

**Reply by fkiraly** [reply]
FYI @chrisholder, @MatthewMiddlehurst, @TonyBagnall

**Reply by cedced19** [reply]
It might be the case. In any case I do not explicitly recognize the algorithms from the references.
